components/providers/AccordionLayout.vue

```diff
@@ -1,72 +1,113 @@
 <template>
   <article class="accordion-layout">
-    <cv-code-snippet
-      kind="oneline"
-      light
-      feedback-aria-label="Copy code snippet"
-    >
-      <!-- TODO: replace w/ dynamic code snippet -->
-      <span>pip install something</span>
-    </cv-code-snippet>
-    <img :src="image" class="accordion-layout__image">
     <p class="accordion-layout__description">
       {{ description }}
     </p>
-    <AppCta
-      kind="ghost"
-      v-bind="cta"
-    />
-    <!-- TODO: replace w/ dynamic code sample -->
-    <cv-code-snippet
-      kind="multiline"
-      light
-      feedback-aria-label="Copy code snippet"
-    >
-      import qiskit
-      from qiskit_runtime import ...
-
-      (very) short code example
-      Lorem ipsum dolor sit amet
-      consectetur adipiscing elit
-      Suspendisse id laoreet urna
-      nec egestas lorem
-      Donec porta
-      mauris sed facilisis pulvinar
-    </cv-code-snippet>
+    <img v-if="image" :src="image" class="accordion-layout__image">
+    <div class="accordion-layout__code-snippet">
+      <code class="accordion-layout__code-cell">
+        <pre class="accordion-layout__code-cell__line">{{ installation }}</pre>
+      </code>
+      <cv-button
+        title="Copy"
+        class="accordion-layout__copy-button"
+        @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} install code`, 'providers')]"
+      >
+        Copy
+      </cv-button>
+    </div>
```

Comment:
I think we can create a component for code snippets. This code is repeated several times (here 2 times and on the new Quick Start another 2 times). This is my suggestion, but please, have in mind that I'm gonna do this code without testing it (so don't merge without checking it):

On a new component:

```suggestion
    <div class="code-snippet">
      <code class="code-snippet__code-cell">
        <pre
          v-for="(line, index) in codeLines"
          :key="index" 
          class="code-snippet__code-cell__line"
          :class="{'code-snippet__code-cell_comment' : line.startsWith('#')}"
          >
            {{ line }}
        </pre>
      </code>
      <cv-button
        title="Copy"
        class="code-snippet__copy-button"
        @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} install code`, <location>)]"
      >
        Copy
      </cv-button>
    </div>
```

Reply:
Great suggestion, updated in https://github.com/Qiskit/qiskit.org/pull/3000/commits/88ac621a65a1abc7d1e8637babbe6b846672e819